server/services/nextcloud-talk/lib/message/message.bot.js

```diff
@@ -0,0 +1,77 @@
+const EventEmitter = require('events');
+const get = require('get-value');
+const logger = require('../../../../utils/logger');
+
+class NextcloudTalkBot extends EventEmitter {
```

Comment:
We don't really use this pattern in Gladys (class with inheritance in one big file)

To stay consistent with the rest of the project, could you do it with a prototype and 1 function per file like on other services?

Reply:
I rewrite full process, it should be better